microapps/ArWorldScaleApp/app/src/main/java/com/arcgismaps/toolkit/arworldscaleapp/screens/MainScreen.kt

```diff
@@ -149,24 +153,42 @@ fun MainScreen() {
 
                 DropdownMenu(
                     expanded = actionsExpanded,
-                    onDismissRequest = { actionsExpanded = false })
-                {
-                    DropdownMenuItem(
-                        text = { Text(stringResource(R.string.world_tracking_dropdown_item)) },
-                        onClick = {
-                            trackingMode = WorldScaleTrackingMode.World()
-                            actionsExpanded = false
-                        }
-                    )
-                    DropdownMenuItem(
-                        text = { Text(stringResource(R.string.geospatial_tracking_dropdown_item)) },
-                        onClick = {
-                            trackingMode = WorldScaleTrackingMode.Geospatial()
-                            actionsExpanded = false
-                        }
-                    )
+                    onDismissRequest = { actionsExpanded = false }
+                ) {
+                    val trackingModes = remember {
+                        listOf(
+                            WorldScaleTrackingMode.World(),
+                            WorldScaleTrackingMode.Geospatial()
+                        )
+                    }
+
+                    trackingModes.forEach { trackingMode ->
+                        DropdownMenuItem(
+                            text = { Text("${trackingMode::class.java.simpleName} tracking") },
+                            onClick = {
+                                selectedTrackingMode = trackingMode
```

Comment:
I suggest we close the drop down menu as soon as a user selects a tracking mode. We already do that for "Privacy Info":
```suggestion
                                selectedTrackingMode = trackingMode
                                actionsExpanded = false
```